utilities.tex

```diff
@@ -563,7 +596,37 @@
 For templates \tcode{greater}, \tcode{less}, \tcode{greater_equal}, and
 \tcode{less_equal}, the specializations for any pointer type yield a total order,
 even if the built-in operators \tcode{<}, \tcode{>}, \tcode{<=}, \tcode{>=}
-do not.
+do not. \ednote{The following sentence is taken from the proposed resolution of
+\href{https://cplusplus.github.io/LWG/lwg-active.html\#2450}{LWG \#2450}.}
+\newtxt{For template specializations \tcode{greater<void>}, \tcode{less<void>},
+\tcode{greater_equal<void>}, and \tcode{less_equal<void>}, if the call operator calls a
+built-in operator comparing pointers, the call operator yields a total order.}
```

Comment:
I'm uncomfortable with this. It gets into platform-y and core-language-y territory. If you feel strongly about this, I suggest you file a library issue and let LWG deal with it.